packages/react-examples/package.json

```diff
@@ -51,7 +51,6 @@
     "@fluentui/react-focus": "^8.1.5",
     "@fluentui/react-hooks": "^8.2.3",
     "@fluentui/react-icons-mdl2": "^1.1.4",
-    "@fluentui/react-make-styles": "^9.0.0-alpha.42",
```

Comment:
allright looks like we'll need to keep this here until all remaining packages are not migrated -> tracking the issue here https://github.com/microsoft/fluentui/issues/18752